nowcast/delphi_nowcast/data_containers.py

```diff
@@ -0,0 +1,95 @@
+"""Data container classes for holding sensor configurations and data needed for fusion."""
+
+from dataclasses import dataclass, field
+from typing import List, Union
+
+from numpy import ndarray, nan, nanmean, isnan
+from pandas import date_range
+
+
+@dataclass(frozen=True)
+class SensorConfig:
+    """Dataclass for specifying a sensor's name, number of lag days, and origin source/signal."""
+
+    source: str
+    signal: str
+    name: str
+    lag: int
+
+
+@dataclass
+class LocationSeries:
+    """Data class for holding time series data specific to a single location."""
+
+    geo_value: str = None
+    geo_type: str = None
+    dates: List[int] = field(default_factory=lambda: [])
```

Comment:
In general, I think we should prefer storing `datetime.date` objects for dates rather than strings or ints.  Given the amount of conversion to date types elsewhere in this code, it seems appropriate here to do so.

Reply:
updated to use dates